hack/udev_v238_sle15-sp3.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+import re
+import os
+import sys
+import shutil
+from datetime import datetime
+from subprocess import check_output, DEVNULL
+
+"/sys/class/net/enp1s0f0/device/modalias"
```

Comment:
according to https://github.com/systemd/systemd/blob/main/src/udev/udev-builtin-net_id.c#L259
It will load modalias file and PCI class code and subclass code will be inside.
just a note for full /sys path

Reply:
If it's just a note, although this string has no effect whatsoever, I'd say using a comment would be better.

Reply:
```suggestion
"""/sys/class/net/enp1s0f0/device/modalias"""
```

python use no effect string to be comment also

Reply:
Indeed, mostly seen in docstrings. Python also uses `#` as comments.

Reply:
```suggestion
# /sys/class/net/enp1s0f0/device/modalias
```